modules/update-mirror-repository-oc-mirror.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="update-mirror-repository_oc_mirror_{context}"]
+= Mirroring images using oc-mirror
+
+Using the oc-mirror OpenShift CLI (`oc`) plug-in is the recommended method for mirroring images onto a mirror registry in a fully or partially disconnected environment.
```

Comment:
nit: do we have other recommended methods for connected environments?  If not (and I'm not sure what they'd be; configuring pass-through mirroring in registry?), we might want to phrase this like:

> is the recommended method for mirroring images onto a mirror registry.  This can be useful in several situations, including fully or partially disconnected environments.

Or something to make it clear that other situations (like "I just don't trust Quay to stay up/reachable/fast-for-my-moon-base/...") can happily use this process too.

Reply:
I'll rephrase the writing to what you suggested in your first comment